src/Raven.Client/Documents/Operations/SchemaValidation/SchemaValidationConfiguration.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.SchemaValidation;
+
+public sealed class SchemaValidationConfiguration
+{
+    private Dictionary<string, SchemaDefinition> _validatorsPerCollection;
+    
+    public bool Disabled { get; set; }
```

Comment:
Is there any way to remove the schema validation? I found only this property to disable it

Reply:
Do we miss `DeleteSchemaValidationOperation`?